queries/escrow/useEscrowDataQueryV1.tsx

```diff
@@ -7,42 +7,38 @@ import QUERY_KEYS from 'constants/queryKeys';
 
 import { isWalletConnectedState, networkState, walletAddressState } from 'store/wallet';
 import { appReadyState } from 'store/app';
+import { EscrowData, Schedule } from 'hooks/useEscrowDataQueryWrapper';
 
-export type EscrowData = {
-	claimableAmount: number;
-	schedule: Schedule;
-	totalEscrowed: number;
-	totalVested: number;
-};
-
-type Schedule = Array<
-	| {
-			quantity: number;
-			date: Date;
-	  }
-	| []
->;
-
-const useEscrowDataQuery = (options?: QueryConfig<EscrowData>) => {
+const useEscrowDataQueryV1 = (options?: QueryConfig<EscrowData>) => {
 	const isWalletConnected = useRecoilValue(isWalletConnectedState);
 	const walletAddress = useRecoilValue(walletAddressState);
 	const network = useRecoilValue(networkState);
 	const isAppReady = useRecoilValue(appReadyState);
 
 	return useQuery<EscrowData>(
-		QUERY_KEYS.Escrow.Data(walletAddress ?? '', network?.id!),
+		QUERY_KEYS.Escrow.DataV1(walletAddress ?? '', network?.id!),
 		async () => {
 			const {
-				contracts: { RewardEscrow },
+				contracts: { RewardEscrow, RewardEscrowV2 },
 				utils: { formatEther },
 			} = synthetix.js!;
 
-			const [accountSchedule, totalEscrowed, totalVested] = await Promise.all([
+			const [
+				accountSchedule,
+				totalEscrowed,
+				totalVested,
+				unformattedTotalBalancePendingMigration,
+			] = await Promise.all([
 				RewardEscrow.checkAccountSchedule(walletAddress),
 				RewardEscrow.totalEscrowedAccountBalance(walletAddress),
 				RewardEscrow.totalVestedAccountBalance(walletAddress),
+				RewardEscrowV2.totalBalancePendingMigration(walletAddress),
```

Comment:
note this has to be loaded with the v1 data since it is a check if we need to use v1 and if we are using v1 then we need this number. If we are not using v1 then we don't care about this number anyways